webpack/assets/javascripts/react_app/components/UserJwt/components/UserJwtForm.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { Grid, Dropdown, MenuItem, Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import { translate as __ } from '../../../common/I18n';
+import {
+  UNIT_NEVER,
+  UNIT_HOURS,
+  UNIT_DAYS,
+  UNIT_WEEKS,
+  UNIT_MONTHS,
+} from '../UserJwtConstants';
+
+const UserJwtForm = ({ handleSubmit }) => {
+  const expirationUnits = [
+    { label: __('Never'), value: UNIT_NEVER },
+    { label: __('Hours'), value: UNIT_HOURS },
+    { label: __('Days'), value: UNIT_DAYS },
+    { label: __('Weeks'), value: UNIT_WEEKS },
+    { label: __('Months'), value: UNIT_MONTHS },
+  ];
+
+  const [expirationUnit, setExpirationUnit] = useState(expirationUnits[0]);
+  const [expirationValue, setExpirationValue] = useState('');
+
+  const changeUnit = u => {
+    if (u.value === UNIT_NEVER) {
+      setExpirationValue('');
+    }
+    setExpirationUnit(u);
+  };
+
+  return (
+    <React.Fragment>
+      <Grid.Row>
+        <Grid.Col>
+          <div className="form-group user-jwt-form">
+            <label className="control-label col-md-2 col-lg-2">
+              {__('Expiration')}
+            </label>
+            <div className="col-md-4 col-lg-2">
```

Comment:
This makes it smaller then before for large screens 
![Screenshot from 2020-09-09 15-00-17](https://user-images.githubusercontent.com/30431079/92595531-31fab780-f2ad-11ea-96cf-345941d67716.png)

Reply:
I redesigned the form, hope now it's better then before.